src/services/codefixes/importFixes.ts

```diff
@@ -445,9 +445,12 @@ namespace ts.codefix {
             const aliased = checker.getImmediateAliasedSymbol(defaultExport);
             return aliased && getDefaultExportInfoWorker(aliased, Debug.assertDefined(aliased.parent), checker, compilerOptions);
         }
-        else {
-            return { symbolForMeaning: defaultExport, name: moduleSymbolToValidIdentifier(moduleSymbol, compilerOptions.target!) };
+
+        if (defaultExport.escapedName !== InternalSymbolName.Default &&
+            defaultExport.escapedName !== InternalSymbolName.ExportEquals) {

```

Comment:
I'd also check that that name `isIdentifierText`, just in case the name comes from, say, a jsdoc typedef or a js property assignment where the original declaration's name wasn't an identifier.

Reply:
@weswigham we don't need to check that since this is the name just to compare the defaultExportSymbol ==== symbol we are looking for and that symbol's name does default to symbol.name if its not `export=` or `default`. The actual text to be displayed is calculated later.